lib/radiator/contribution/podcast_contribution.ex

```diff
@@ -15,6 +16,25 @@ defmodule Radiator.Contribution.PodcastContribution do
   @doc false
   def changeset(contribution, attrs) do
     contribution
-    |> cast(attrs, [:position])
+    |> cast(attrs, [:position, :person_id, :role_id, :podcast_id])
+    |> foreign_key_constraint(:person)
+    |> foreign_key_constraint(:role)
+    |> foreign_key_constraint(:podcast)
+    |> ensure_position()
+  end
+
+  def ensure_position(changeset) do
```

Comment:
So you're WET here, not DRY? ;)

Reply:
yeah currently. As maybe this could be also just a default from the db in the migration? didn't have the time to choose the right one, so did it this way.